author/topics/document-format/text.adoc

```diff
@@ -274,6 +274,32 @@ from the image.
 image::logo.jpg
 --
 
+=== Captions and titles
+
+As elsewhere in Metanorma, the caption of an image (in Metanorma, of the figure containing the image)
+is given with a line prefixed with dot. In Asciidoctor, the title attribute is treated as identical
+to the dot-prefixed caption:
+
+[source,asciidoctor]
```

Comment:
There is `[source,adoc]` but not `[source,asciidoctor]`. Please help update (all other instances have been fixed).

Reply:
Updated PR to change all instances